Support shift operations in serialize/export

minjang opened the pull request:
Summary: Support shift operators in `_SYM_OPS`.

Differential Revision: D87157915

shawnxu26 reviewed (approved):
LGTM. Happy New Year!

minjang committed:
Support shift operations in serialize/export (#167913)

Summary:

Support shift operators in `_SYM_OPS`.

Reviewed By: XueningXu

Differential Revision: D87157915

atalman commented:
@pytorchbot merge -f "error is not related"
